integration/tests/main/03-create-private-package-repository.spec.js

```diff
@@ -0,0 +1,130 @@
+const { test, expect } = require("@playwright/test");
+const { KubeappsLogin } = require("../utils/kubeapps-login");
+const utils = require("../utils/util-functions");
+
+test("Create a new private package repository successfully", async ({ page }) => {
+  test.setTimeout(120000);
```

Comment:
I don't see this timeout in the old test? If it's necessary, just provide a comment explaining where the issue is so it can be investigated more easily later?

Reply:
This does not set a timeout to be effective at that point. It just sets the [timeout for this single test](https://playwright.dev/docs/test-timeouts#set-timeout-for-a-single-test). If the test takes more than 120 seconds, it will be considered as failed. There is a default that can be set on Playwright settings.